### PR TITLE
feat[SC-1]: KFC- India - Sign page - User should able to skip the sign In

### PR DESCRIPTION
**Pull request by Nikhil-CES92**
### Summary
This PR contains Playwright scripts for the Sign page, specifically for skipping the sign-in functionality.

### Links: -
Jira: - n/a
Test case: - n/a

### Changes/Note: -

1. Implemented a project-level POM framework folder structure to enhance code reusability and maintainability.  
2. Created two utility functions for assertions and interactions.  
3. Added a page-level class to store functions for page-specific operations.
4. Currently, some configurations in the config file are commented out, but they will be configured eventually.

### CODE REVIEW

File: tests/signInPageTests.spec.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import { test } from '@playwright/test'
+import HomePage from '../PageObjects/homePageFns'
+import Verify from '../Utils/Assertions/verify'
+import Click from '../Utils/Interaction/click'
+import pageSels from '../Utils/Selectors/pageSels'
+
+const { skipBtnText } = pageSels.signInPage
```
Comment (ajinkyaces) [inline tests/signInPageTests.spec.js]: change the class name it should have Pascal case

File: tests/signInPageTests.spec.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    })
+
+    test('[SC-1]: User should be able to skip the login', async () => {
+        //clicking on signin btn and then skip sign in btn
+        await click.on(signInBtn)
+        await click.onContainText(skipBtnText)
+
+        //verifying the sign is skipped
+        urlCheck = await verify.theUrl()
+        await urlCheck.equalsTo('https://online.kfc.co.in/');
```
Comment (ajinkyaces) [inline tests/signInPageTests.spec.js]: can verify with config file base url

**Review by ajinkyaces — APPROVED** [review]

File: tests/signInPageTests.spec.js (hunk 1 above)
Comment (Nikhil-CES92) [reply tests/signInPageTests.spec.js]: Okay, will update that.

File: tests/signInPageTests.spec.js (hunk 2 above)
Comment (Nikhil-CES92) [reply tests/signInPageTests.spec.js]: Okay will update that.

**Pull request merged**